### PR TITLE
Menu screen view

### PR DESCRIPTION
**Pull request by Maisergodt**
# Background
Relevant for #12 but also based on #11 

# Solution
Created a simple menu view for displaying menu options such as play, settings and exit. Temporarly added a plain Skin for the buttons, perhaps someone with better design skills than me should be assigned to this job. 

Since we probably are going to have a settings menu with centered buttons of some sort, I was thinking of having a separate method that will handle this. In that case I need some time to figure out how to do it cleanly and as well implement the settings view.

### CODE REVIEW

**Review by sigridge — APPROVED** [review]
I think this looks really good! I ran it and i really liked the design 😍

**Review by VildeStabell — CHANGES_REQUESTED** [review]
Please add documentation comments before classes, and complex methods. 

Also when I try to run android it only shows in the bottom left corner. Might be my settings, so please check for yourself.

File: android/assets/ui/plain_james.json [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+		"titleFontColor": "black"
+	},
+	"dialog": {
+		"background": "window",
+		"titleFont": "font",
+		"titleFontColor": "black",
+		"stageBackground": "white"
+	}
+}
+}
```
Comment (sigridge) [inline android/assets/ui/plain_james.json]: missing newline

**Comment by Maisergodt** [comment]
> Please add documentation comments before classes, and complex methods.
> 
> Also when I try to run android it only shows in the bottom left corner. Might be my settings, so please check for yourself.

The scaling issues should be fixed now regardless of screen sizes but you'll never if it has introduced new bugs..

**Comment by Maisergodt** [comment]
Regarding testing, I am not sure how to test the classes beside writing unit tests for the scaling and centering method which essentially don't belong to the screen classes. I might have to decouple it such that we follow the modifiability tactic like `loose coupling`  and `reduce size of module`. This will however not have any impact on the functionality. 

A suggestion is to merge these PRs such that dependent views can extend from it while I can assign myself to a new issue to fix it :)

File: desktop/src/com/mygdx/game/desktop/DesktopLauncher.java [hunk 2]
```diff
@@ -8,5 +8,6 @@
 	public static void main (String[] arg) {
 		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
 		new LwjglApplication(new Brainstorming(), config);
+		System.out.println(config.height + " " + config.width);
```
Comment (sigridge) [inline desktop/src/com/mygdx/game/desktop/DesktopLauncher.java]: Is this supposed to be here?

**Review by sigridge — APPROVED** [review]
This looks good to me!

File: desktop/src/com/mygdx/game/desktop/DesktopLauncher.java (hunk 2 above)
Comment (Maisergodt) [reply desktop/src/com/mygdx/game/desktop/DesktopLauncher.java]: Nope, just a debug line I forgot to remove ...

**Comment by VildeStabell** [comment]
> The scaling issues should be fixed now regardless of screen sizes but you'll never if it has introduced new bugs..

It is bigger now, but still looks weird for me. I'll put a picture of how it looks in the bugs slack channel.

**Pull request merged**